pkg/cmd/server/api/validation/validation.go

```diff
@@ -271,9 +271,11 @@ func ValidateLDAPSyncConfig(config api.LDAPSyncConfig) ValidationResults {
 		validationResults.AddWarnings(configResults.Warnings...)
 		numConfigs++
 	}
-	if numConfigs != 1 {
-		validationResults.AddErrors(fielderrors.NewFieldInvalid("", config.LDAPSchemaSpecificConfig,
-			"only one schema-specific config is allowed"))
+	if numConfigs > 1 {
+		validationResults.AddErrors(fielderrors.NewFieldInvalid("", config, "only one schema-specific config is allowed"))
```

Comment:
add the names to a list, and include the list in the error if len(configNames) > 1, `...found %v", configNames)`, etc